docs/pages/blog/2023-discord-announcement.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'MUI is now on Discord!'
+description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
+date: 2023-08-02T00:00:00.000Z
+authors: ['richbustos']
+tags: ['News']
+card: true
+---
+
+<a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1280" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>
+
+Creating a place where we can interact with our community has been our priority for this year.
+Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/MUI_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).
+
+That's why we have officially launched our [Discord Server](https://mui.com/r/discord/) and would love for you to be part of our community!
+
+Come join to engage in lively discussions, share your projects, and interact with the MUI team & others!
+
+## Why you should join our Discord Server?
+
+There are a plethora of reasons why you should join, but ultimately, it came down to 3 things:
+
+1. **Real-time communication with our community**:
+   As mentioned above, our communication with the community was quite slow.
+   Members would either file an issue on Github, tweet us, or send us a message directly on our website.
```

Comment:
"Members" feels like an odd way to describe developers who use MUI products. 

```suggestion
   Developers are frequently unsure whether to open an issue on Github, tweet at us, or send us an email when they have a question or want to share feedback.
```

Reply:
ahh yeah, I get your point. Was hesitant to specify "developers" because there could also be designers or product managers who use our products.

I guess in a sense, those roles are also developers. 

Will change all "members" to "developers"